docker-compose.yml

```diff
@@ -23,7 +23,7 @@ services:
       - redis-data:/data
       - ./data/redis/conf/redis.conf.template:/usr/local/etc/redis/redis.conf.template
       - ./data/certbot/conf/:/etc/letsencrypt
-    command: /bin/sh -c "sed -e 's/$$HOSTNAME/redis.${DOMAIN_NAME:-xlab.blindside-dev.com}/' /usr/local/etc/redis/redis.conf.template > /usr/local/etc/redis/redis.conf && exec redis-server --appendonly yes"
+    command: /bin/sh -c "sed -e 's/$$HOSTNAME/redis.${URL_HOST:-xlab.blindside-dev.com}/' /usr/local/etc/redis/redis.conf.template > /usr/local/etc/redis/redis.conf && exec redis-server --appendonly yes"
```

Comment:
I don't see `DOMAIN_NAME` in the dotenv or anywhere else in the code.
I believe we can strictly use `URL_HOST` as it is supposedly pointing to the same data, and gets rid of an extra variable - please correct me if I am wrong.

Reply:
No, it is the opposite. We use DOMAIN_NAME because in v2 we now may have different applications under the same domain. So, in all case, we add the variable to the `dotenv` file